cfpb/capital-framework
# Gh-pages docs: fix font path for cf-icons

Fixes gulp config to put the fonts folder where Capital Framework cf-icons component and documentation expects them to be.

## Changes

- updated gulp/config.js with the correct file path
- re-compiled with `npm run build`

## Testing

- check out branch
- `npm start`
- Confirm icons work on [cf-notification](http://localhost:3000/components/cf-notification/) and [cf-icons](http://localhost:3000/components/cf-icons/) docs - they shouldn't 404 anymore.

## Screenshots
![screen shot 2017-09-06 at 5 30 54 pm](https://user-images.githubusercontent.com/702526/30135551-359d5258-9329-11e7-8871-968cf3ede2f1.png)
![screen shot 2017-09-06 at 5 30 45 pm](https://user-images.githubusercontent.com/702526/30135550-359c3d6e-9329-11e7-87f8-81e0bbe0e226.png)


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers

- [ ] Blackberry Bold

### Accessibility


### Other
